### ISSUE TITLE
Youtube videos extremely blurred and impossible to watch

### ISSUE THREAD

**Issue by Hallilogod** [zen-browser/desktop#1571]
### Captchas

- [X] I have read the instructions.
- [X] I have searched existing issues and avoided creating duplicates.
- [X] I am not filing an enhancement request.

### What happened?

I can't watch any YouTube video on Zen, it's EXTREMELY blurred and it's impossible to see anything. However, this does not always happen; sometimes it works, sometimes it doesn't. I got it working right now by using Picture-in-Picture mode and disabling it again. Once, I also got it working by changing my privacy setting from strict to standard, which I thought would have been the problem, but it randomly reappeared. I have no idea what causes this. I tried disabling all extensions, no difference.

![image](https://github.com/user-attachments/assets/ab989fba-c717-4c4e-b7b1-0f2c10a1802d)

### Reproducible?

- [X] I have checked that this issue cannot be reproduced on Mozilla Firefox.

### Version

1.0.1-a.2 (64-bit)

### What platform are you seeing the problem on?

Windows

**Comment by mauro-balades** [comment]
What if you do a hard reload? CTRL+R

**Comment by Hallilogod** [comment]
Tried, doesn't fix it.

**Comment by Hallilogod** [comment]
Im pretty sure i also tried restarting the browser, what i didn't try yet is creating a new profile and checking if the problem is there too. The problem is that this randomly happens and i don't know what causes it.

**Comment by mauro-balades** [comment]
Try disabling tracking protection

**Comment by Hallilogod** [comment]
Tried that too

**Comment by Hallilogod** [comment]
Ill try more if it happens again

**Comment by onthejobtraining** [comment]
Yeah, use a separate profile for youtube for a few days entirely and see if the issue occurs because this seems like an issue specific to you.

**Comment by Hallilogod** [comment]
Its happening again, im trying a new profile now, however i really didnt change many settings in zen. ill try tho

![image](https://github.com/user-attachments/assets/315d5966-b4ec-421e-9b05-e0a719f0a9b9)

**Comment by Hallilogod** [comment]
I also noticed some other UI problems with zen, its a really great browser but currently seems very unstable

**Comment by NOCanoa** [comment]
> What if you do a hard reload? CTRL+R @mauro-balades 
hard reload is CTRL + SHIFT +R

**Comment by Hallilogod** [comment]
Just tried with a freshly new created profile, same thing. Reloading with either ctrl r or ctrl shift r fixes nothing.

**Comment by Hallilogod** [comment]
I dont think anyone will be able to help me, zen is great but seems unusable because of a few issues

**Comment by onthejobtraining** [comment]
I guess so, what extensions and settings are you using on the new profile that you created? did you try default settings with no extension (or maybe just uBO)

**Comment by Hallilogod** [comment]
![image](https://github.com/user-attachments/assets/cb9867b3-6e56-436b-aea9-814d26c5ae1e)
Just did a fresh windows reinstall, same issue, new profile, also tried the generic version of zen.

**Comment by Hallilogod** [comment]
![image](https://github.com/user-attachments/assets/942e121c-9e31-46f2-af97-3c3a1307cbe2)
when opening this account side panel it works for some reason, maybe that info can help debugging

**Comment by RayZ3R0** [comment]
Does this still happen on latest version? (a.14 as of this comment)

**Comment by Hallilogod** [comment]
I can try it again

**Comment by salamijason** [comment]
Hi did you try it again?

**Comment by Hallilogod** [comment]
Currently i don't have my laptop which is the only device where it happens on, i can try in around 1 - 2 weeks.

**Comment by Hallilogod** [comment]
Hey, i didn't got the problem again so it might be fixed? I'd still let this issue open for some weeks and if it never happens again i'm gonna leave another comment here.

**Comment by salamijason** [comment]
Hey! I'll close it for now since we're tryna clean up the issues. Please reopen this if you encounter the issue again!

**Comment by Hallilogod** [comment]
Yup it seems to be fixed, forgot to close it!
